met-web/src/components/engagement/form/AddSurveyBlock.tsx

```diff
@@ -96,7 +96,7 @@ export const AddSurveyBlock = () => {
                                 <MetWidget
                                     key={survey.id}
                                     title={survey.name}
-                                    onEditClick={() => navigate(`/survey/build/${survey.id}`)}
+                                    onEditClick={() => navigate(`/survey/:${survey.id}/build`)}
```

Comment:
oh I don't think you need to add ':' when you navigate. When you create that route you add it to note that whatever comes after the : is the name of the variable. When you navigate to it you just pass the variable.